client/src/competition/competition-details-modal/Modal.style.js

```diff
@@ -0,0 +1,70 @@
+import styled from 'vue-styled-components';
+
+export const ModalWrapper = styled('article')`
```

Comment:
It would be nice to have some generic Modal component

Reply:
Very good point, I have one suggestion regarding that, but I would like to discuss it on the coding day.

Reply:
ok, but we have Modal in our task as well
could you review https://github.com/extreme-startup/questionator/pull/64 then?

Reply:
Done :)